dashboard/settings.py

```diff
@@ -190,80 +191,84 @@
 
 #Shib
 
-SAML2_URL_PATH = '/accounts/'
-# modify to use port request comes
-SAML2_URL_BASE = getenv('DJANGO_SAML2_URL_BASE', 'http://localhost:5001/accounts/')
-
-INSTALLED_APPS += ('djangosaml2',)
-AUTHENTICATION_BACKENDS = (
-    'djangosaml2.backends.Saml2Backend',
-)
-LOGIN_URL = '%slogin/' % SAML2_URL_PATH
-SESSION_EXPIRE_AT_BROWSER_CLOSE = True
-
-BASEDIR = path.dirname(path.abspath(__file__))
-SAML2_FILES_BASE = os.environ.get('SAML2_FILES_BASE', '/saml/')
-
-SAML_CONFIG = {
-    'xmlsec_binary': '/usr/bin/xmlsec1',
-    'entityid': '%smetadata/' % SAML2_URL_BASE,
-
-    # directory with attribute mapping
-    # 'attribute_map_dir': path.join(BASEDIR, 'attribute-maps'),
-    'name': 'Student Dashboard',
-    # this block states what services we provide
-    'service': {
-        # we are just a lonely SP
-        'sp': {
-            'name': 'Student Dashboard',
-            'name_id_format': ('urn:oasis:names:tc:SAML:2.0:'
-                               'nameid-format:transient'),
-            'authn_requests_signed': 'true',
-            'allow_unsolicited': True,
-            'endpoints': {
-                # url and binding to the assetion consumer service view
-                # do not change the binding or service name
-                'assertion_consumer_service': [
-                    ('%sacs/' % SAML2_URL_BASE, saml2.BINDING_HTTP_POST),
-                ],
-                # url and binding to the single logout service view+
-
-                # do not change the binding or service name
-                'single_logout_service': [
-                    ('%sls/' % SAML2_URL_BASE, saml2.BINDING_HTTP_REDIRECT),
-                    ('%sls/post' % SAML2_URL_BASE, saml2.BINDING_HTTP_POST),
-                ],
+if getenv_bool('STUDENT_DASHBOARD_SAML', 'false'):
+    
+    import saml2
+
+    SAML2_URL_PATH = '/accounts/'
+    # modify to use port request comes
+    SAML2_URL_BASE = getenv('DJANGO_SAML2_URL_BASE', 'http://localhost:5001/accounts/')
+
+    INSTALLED_APPS += ('djangosaml2',)
+    AUTHENTICATION_BACKENDS = (
+        'djangosaml2.backends.Saml2Backend',
+    )
+    LOGIN_URL = '%slogin/' % SAML2_URL_PATH
+    SESSION_EXPIRE_AT_BROWSER_CLOSE = True
+
+    BASEDIR = path.dirname(path.abspath(__file__))
+    SAML2_FILES_BASE = os.environ.get('SAML2_FILES_BASE', '/saml/')
+
+    SAML_CONFIG = {
+        'xmlsec_binary': '/usr/bin/xmlsec1',
+        'entityid': '%smetadata/' % SAML2_URL_BASE,
+
+        # directory with attribute mapping
+        # 'attribute_map_dir': path.join(BASEDIR, 'attribute-maps'),
+        'name': 'Student Dashboard',
+        # this block states what services we provide
+        'service': {
+            # we are just a lonely SP
```

Comment:
-:)

Reply:
Ya, I just intended this, these comments were already there ;)